spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -100,15 +102,47 @@ def get_message(self):
 
         return message
 
+    @staticmethod
+    def get_tags(message):
```

Comment:
**suggestion**: We don't have any tests for this action. It would be good to have at least some tests for `get_tags` method.

Reply:
I don't think we have tests for anything in this module, what do you suggest?